docs/database-schema.md

```diff
@@ -0,0 +1,13 @@
+# Database Schema
+
+### wp_notifications table
+* id (int) 11 auto_increment `ID of the notification.`
+* sender `Sender of the notification.`
+* timestamp `Timestamp of when the notification was triggered.`
+* recipients `Collection of notification recipients.`
+* title_key `Notification title key.`
+* message_key `Notification message key.`  (optional)
```

Comment:
Maybe this two fields should be coupled in the Message model.

Reply:
Is there an open issue discussing this model?

Reply:
Not yet.

Reply:
@raaaahman if it's ok with you, let's merge this for now (as it's not set in stone) and we can iterate if things changed, based on the outcome of the message model discussion.

The main reason for this is that I can see the message data requirement will need to be both somehow stored by both the plugin and theme, but also needing to be dynamically generated as needed.